src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './Header.css';
+import { Link } from 'react-router-dom';
+
+const Header = () => (
+  <div className="Header">
+    <div style={{
+      display: 'flex', alignItems: 'center', maxWidth: '700px', marginLeft: 'calc(100% - 970px)',
+    }}
+    >
```

Comment:
- [ ] It would be better if you delete the inline styles and write them in the `Header.css`, Just give it a clasName and write the same properties. This would help you to have an organized project. 
😄 👍